packages/suite/src/components/wallet/CoinjoinSummary/BalanceSection.tsx

```diff
@@ -75,6 +77,7 @@ export const BalanceSection = ({ account }: BalanceSectionProps) => {
             <AnonymizeButton
                 onClick={goToSetup}
                 icon="ARROW_RIGHT_LONG"
+                isDisabled={isTorDisabled}
```

Comment:
button was not disabled before? was it intentional or bug?

Reply:
@hynek-jina @komret  is it fine that the "Anonymize" button  is disable when Tor is not enable so the user cannot start coinjoin until Tor is enable? Or should it be different as when the user does not have coins in the account?

![image](https://user-images.githubusercontent.com/5362163/202697414-388e6ba2-96dd-49d8-985f-0ae0996bf1fd.png)

Reply:
Yes, button should be disabled..

Reply:
Here is related issue: https://github.com/trezor/trezor-suite/issues/6908